MODULE.bazel

```diff
@@ -14,14 +14,12 @@ bazel_dep(name = "rules_gazebo", version = "0.0.2")
 bazel_dep(name = "gz-utils")
 bazel_dep(name = "gz-math")
 
-archive_override(
+local_path_override(
     module_name = "gz-utils",
-    strip_prefix = "gz-utils-gz-utils3",
-    urls = ["https://github.com/gazebosim/gz-utils/archive/refs/heads/gz-utils3.tar.gz"],
+    path = "../gz-utils",
 )
 
-archive_override(
+local_path_override(
     module_name = "gz-math",
-    strip_prefix = "gz-math-gz-math8",
-    urls = ["https://github.com/gazebosim/gz-math/archive/refs/heads/gz-math8.tar.gz"],
+    path = "../gz-math",
```

Comment:
I think the changes to this file should be reverted

Reply:
Removed, thanks!